driver/docker/docker.go

```diff
@@ -301,9 +302,35 @@ func (d *Driver) fetchOutputs(ctx context.Context, container string, op *driver.
 		return opResult, err
 	}
 
+	// if an applicable output is expected but does not exist and it has a
+	// non-empty default value, create an entry in the map with the
+	// default value as its contents
+	for name, output := range op.Bundle.Outputs {
```

Comment:
One part that I'm unsure about is whether or not it is sufficient to just write to the output map? The spec says

> A runtime can leverage appropriate in-memory volume mounted at the path location for storing these outputs.

If we just write to the map, then the mounted volume will be missing any outputs that have a default, and only the claim (which this map is written) will have the full set of outputs. Having said that I'm not sure where the best place would be to write out the files with the defaults so that they both agree however.

Thoughts?

Reply:
Hmmm, good point.  I'm also unclear on how to address this.

It feels like the outputs written -- or not written -- to the container's filesystem would be under the bundle author's domain and so my first thought is that it'd be odd for the library to write missing/additional files there.  Instead, adding/supplying default values to the operation/claim as a runtime responsibility seems more natural.  However, this is just my initial take and is moot if it differs from the spec.